lms/djangoapps/certificates/queue.py

```diff
@@ -214,6 +214,18 @@ def add_cert(self, student, course_id, course=None, forced_grade=None, template_
         Returns the newly created certificate instance
         """
 
+        if not hasattr(course_id, 'ccx'):
```

Comment:
That's exactly right, that `not` shouldn't be there and is why a bunch of tests are failing in this really silly way:

```
Cannot create certificate generation task for user 2 in the course 'org.188/course_188/Run_188'; certificates are not allowed for CCX courses.
```

I've fixed this, thanks. :)